packages/fxa-payments-server/src/lib/amplitude.ts

```diff
@@ -70,18 +71,32 @@ export function addGlobalEventProperties(props: GlobalEventProperties) {
 
 export function subscribeToReduxStore(store: Store) {
   let unsubscribe: ReturnType<typeof store.subscribe>;
-  const uidObs = () => {
+  const obs = () => {
     const profile = selectors.profile(store.getState());
+    const subscriptions = selectors.subscriptions(store.getState());
```

Comment:
Good idea to look and see how some of these existing fields were getting picked up already. Seems reasonable to me for SubPlat 2.5.